### DISCUSSION TITLE
Implementing Identify API

### DISCUSSION THREAD

**Discussion by marlocorridor** [abr4xas/clarity-laravel#5] [Ideas]
Hi, let us add an option to show the record the user's identification.

https://learn.microsoft.com/en-us/clarity/setup-and-installation/identify-api#storing-and-managing-custom-ids

**Comment by marlocorridor** [comment]
Here is my draft

https://github.com/abr4xas/clarity-laravel/commit/392042c477ca9de529b3aff33962951680672735

**Reply by abr4xas** [reply]
Hey @marlocorridor send your PR 😃

**Reply by marlocorridor** [reply]
on it's way 😁

**Reply by marlocorridor** [reply]
Hi @abr4xas! 
Here is our pull request #6

**Reply by abr4xas** [reply]
Merged 🚀 🚀
